### PR TITLE
Theo/initialisation

### PR DESCRIPTION
**Pull request by Theo-Reignier**
### Purpose

Integration of 3 'initialization' nodes:
- Quantum Circuit (can be set to make use of registers or simply qubits and cbits)
- Quantum Register
- Classical Register

This makes use of 2 global variables:
- script
- quantumCircuit

I will need to add error handling, comments and other minor changes on these nodes.
However, since this is a working version, I merge it with the master branch so that everyone can use it to develop their own nodes.

### CODE REVIEW

**Review by KhesimReid — APPROVED** [review]
Nice work, Theo!

**Comment by louislefevre** [comment]
I'll review in a couple hours 👍

**Comment by louislefevre** [comment]
@Theo-Reignier I just pushed some linting/refactoring commits (using `npm run lint` command). The functionality all still works the same (I tested it using the examples you posted in Teams), though you should probably pull the changes and run it yourself to make sure before merging.

Other than that, LGTM!

File: package-lock.json [hunk 1]
```diff
@@ -5,7 +5,6 @@
   "requires": true,
   "packages": {
     "": {
-      "name": "node-red-contrib-quantum",
       "version": "0.0.1",
```
Comment (louislefevre) [inline package-lock.json:8]:
```suggestion
      "name": "node-red-contrib-quantum",
      "version": "0.0.1",
```
Did you intend to remove this? I ran `npm i --package-lock-only` and it re-added it, so I think it must be required.

Comment (Theo-Reignier) [reply package-lock.json:8]: That is an error, I did not intend to remove it. Thank you !

**Review by louislefevre — APPROVED** [review]

**Review by KhesimReid — APPROVED** [review]
Looks good to me!

**Comment by louislefevre** [comment]
@KhesimReid Could you submit a new review please? Then we can merge

**Pull request merged**